src/Microsoft.AspNetCore.Server.Kestrel/Http/Connection.cs

```diff
@@ -31,7 +31,7 @@ public class Connection : ConnectionContext, IConnectionControl
         private ConnectionFilterContext _filterContext;
         private LibuvStream _libuvStream;
         private FilteredStreamAdapter _filteredStreamAdapter;
-        private Task _readInputContinuation;
+        private Task _readInputTask;
```

Comment:
Having "continuation" is the name is needlessly leaking information about what's happening inside `ReadInput()`.